the job is done (replace and search)

malzafaker opened the pull request.

7R41N33 committed:
7R41N33: swap and search

search refactoring

rubocop refactoring

swap refactoring

remove dumps

Added one more assert to Hitin suite

Update .travis.yml

Added one more test case to lks007 suite.

search and replace methods were implemented

pull request comments fixes

Complete 2 task

add rubocop and docker

run rubocop through bundler

not require rubocop in gemfile

typo fix in travis config

New tasks

Linted

Styles linted

Update README.md

rubocop refactoring

swap refactoring

swap: refactoring

Makapoxa committed:
add binsearch

Makapoxa committed:
add minimal chain length finder

malzafaker committed:
ready test_replace

malzafaker committed:
ready test_replace

malzafaker committed:
ready search

malzafaker committed:
ready search

malzafaker committed:
little fix

malzafaker committed:
little fix

malzafaker committed:
added sort array

malzafaker committed:
added sort array

artempartos commented:
rebase it

artempartos commented on test/vireasss/arrays/test.rb:
```diff
@@ -5,8 +5,8 @@ class Vireasss::ArraysTest < Minitest::Test
   def test_replace
-    skip
-    array = [ 3, 2, -8, 4, 1, -6, 7, 8, -99 ]
+    # skip
```
тоже самое - комментарии в репозитории

artempartos commented on test/vireasss/arrays/solution.rb:
```diff
@@ -2,12 +2,28 @@ module Vireasss
     class << self
       def replace(array)
-        array
+        new_array = array.map { |e| (e < 0 ? e = array.min : e ) }
```
все аналогично
https://github.com/Restream/study/pull/7#discussion_r65069336

Andrew8xx8 committed:
New tasks

Andrew8xx8 committed:
Linted

Andrew8xx8 committed:
Styles linted

artempartos committed:
Update README.md

Makapoxa committed:
improve chain search algorythm

Andrew8xx8 committed:
123 file added

Andrew8xx8 committed:
Revert "123 file added"

This reverts commit 841eb034dd0a5b8e88b5246e1d61f8c7983f9d23.

artempartos committed:
Merge pull request #6 from Makapoxa/master

arrays challenges

LKS007 committed:
Max_repeat and bin_search after rubycop refact

Seruojka committed:
added solution for test

Seruojka committed:
Removed coverage

Andrew8xx8 committed:
Merge pull request #12 from vasilevsergey/master

added solution for test

Commit:
Added one more test case to lks007 suite

artempartos committed:
Merge pull request #3 from 7R41N33/master

7R41N33: swap and search

Andrew8xx8 committed:
Test result fixed

Andrew8xx8 committed:
Tests fixed

Andrew8xx8 committed:
Merge pull request #11 from LKS007/master

Max_repeat and bin_search after rubycop refact

nmorozov committed:
Average rating method was implemented. Find letter in film name method was implemented

nmorozov committed:
letter scan condition was changed to lower case letter only

artempartos committed:
Merge pull request #16 from 7R41N33/master

fp: rating and chars_count

Andrew8xx8 committed:
Test fixed

artempartos committed:
Merge pull request #15 from nmorozov/master

Average rating method was implemented. Find letter in film name metho…

amxsv committed:
Implement rating and chars_count methods

malzafaker committed:
fix test №1 and add update branch

malzafaker committed:
fix test №1 and add update branch

malzafaker committed:
dont know

malzafaker committed:
dont know №2

artempartos committed:
Merge pull request #17 from amxsv/master

Implement rating and chars_count methods

artempartos committed:
Merge pull request #18 from LKS007/master

fp exercise completed

malzafaker committed:
ready to 90% - rating, have little errors

malzafaker committed:
ready to 90% - rating, have little errors

LKS007 committed:
fp exercise completed

artempartos committed:
add fp2

7R41N33 committed:
fp: rating and chars_count

Test result fixed

Tests fixed

rubocop refactoring

rubocop refactoring

fp: refactoring rating

artempartos committed:
Merge pull request #20 from artempartos/master

add fp2

malzafaker committed:
update branch

malzafaker committed:
delete commit in code - fp
